sources/web/datalab/auth.ts

```diff
@@ -146,8 +146,8 @@ function persistCredentials(tokens: any): string {
 }
 
 export function isSignedIn(): boolean {
-  // If not local, then we should have service account available so consider that signed in.
-  return (process.env.DATALAB_ENV != 'local' || fs.existsSync(appCredFile));
+  var gcloudAccount:string = getGcloudAccount();
```

Comment:
You're right; the 'gcloud config' entry seems to be disjoint from login status.

I'll fix that to use 'gcloud auth list' instead.